Sources/LiftBottomScrollView.swift

```diff
@@ -0,0 +1,52 @@
+import UIKit
+
+class LiftBottomScrollView: UIScrollView {
+
+    var bottomViewControllers: [UIViewController]? {
+        didSet {
+            guard let bottomViewControllers = self.bottomViewControllers else { return }
+            self.addBottomViewControllersAndConstraints(bottomViewControllers)
+        }
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func addBottomViewControllersAndConstraints(_ bottomViewControllers: [UIViewController]) {
+        for (index, viewController) in bottomViewControllers.enumerated() {
+            viewController.view.translatesAutoresizingMaskIntoConstraints = false
+            self.addSubview(viewController.view)
+
+            let firstViewController = index == 0
+            let lastViewController = index == bottomViewControllers.count - 1
+            let middleViewController = !firstViewController && !lastViewController
```

Comment:
These booleans could read much better if they were `isFirstViewController`, `isLastViewController` and so on.